### DISCUSSION TITLE
Replace webrtc with silero for voice detection

### DISCUSSION THREAD

**Discussion by MattBrown88** [KoljaB/RealtimeSTT#68] [Ideas]
Currently, the code uses webrtc to detect voice activity. However, I've found that Silero is much better at filtering out background noise and it does not seem noticeably slower. I'm wondering if we should replace webrtc with Silero in the code shown below. Or give users the option to choose with VAD method to use. 

Thoughts?

```python
if self.stop_recording_on_voice_deactivity:
      if not self._is_webrtc_speech(data, True):
```

**Comment by KoljaB** [comment]
RealtimeSTT already uses both. It wouldn't work that well if I restricted to only one of them.
